newIDE/app/src/AssetStore/ExampleStore/index.js

```diff
@@ -138,28 +239,49 @@ export const ExampleStore = ({
             >
               <ListSearchResults
                 disableAutoTranslate // Search results text highlighting conflicts with dom handling by browser auto-translations features. Disables auto translation to prevent crashes.
-                onRetry={fetchExamplesAndFilters}
-                error={error}
-                searchItems={
-                  searchResults &&
-                  prepareExamples(searchResults.map(({ item }) => item))
-                }
-                getSearchItemUniqueId={getExampleName}
-                renderSearchItem={(exampleShortHeader, onHeightComputed) => (
-                  <ExampleListItem
-                    isOpening={isOpening}
-                    onHeightComputed={onHeightComputed}
-                    exampleShortHeader={exampleShortHeader}
-                    matches={getExampleMatches(exampleShortHeader)}
-                    onChoose={() => {
-                      sendExampleDetailsOpened(exampleShortHeader.slug);
-                      setSelectedExampleShortHeader(exampleShortHeader);
-                    }}
-                    onOpen={() => {
-                      onOpen(exampleShortHeader);
-                    }}
-                  />
-                )}
+                onRetry={fetchGameTemplatesAndExamples}
+                error={gameTemplateStoreError || exampleStoreError}
+                searchItems={searchItems}
+                getSearchItemUniqueId={getItemUniqueId}
+                renderSearchItem={(item, onHeightComputed) => {
+                  if (item.authorIds) {
+                    // This is an exampleShortHeader.
+                    return (
+                      <ExampleListItem
+                        isOpening={isOpening}
+                        onHeightComputed={onHeightComputed}
+                        exampleShortHeader={item}
+                        matches={getExampleShortHeaderMatches(item)}
+                        onChoose={() => {
+                          sendExampleDetailsOpened(item.slug);
+                          onSelectExampleShortHeader(item);
+                        }}
+                        onOpen={onOpenNewProjectSetupDialog}
+                      />
+                    );
+                  }
+                  if (item.listing) {
+                    // This is a privateGameTemplateListingData.
```

Comment:
Same as before, this is a type:
```suggestion
                    // This is a PrivateGameTemplateListingData.
```